docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -542,9 +542,15 @@ To migrate content from one collection to another, if the collections are parts
 #. Deprecate the module/plugin with ``removal_version`` scheduled for the next major version in ``meta/runtime.yml`` of the source collection. The deprecation must be released after the copied content has been included in a release of the target collection.
 #. When the next major release comes:
```

Comment:
Which `next major release` is this talking about?

Reply:
```suggestion
#. When the next major release of the old collection is prepared:
```

Reply:
@dericcrago thanks for the question! @felixfontein thanks for clarifying! committed